### DISCUSSION TITLE
Welcome to OpENer Discussions!

### DISCUSSION THREAD

**Discussion by MartinMelikMerkumians** [EIPStackGroup/OpENer#366] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by balaji-6975** [comment]
hi  [CapXilinx](https://github.com/CapXilinx);
I had  an issue with I/O communication in OpENer c++ ,please guide me to Implement sending and receiving data with plc(.eds File) with sample code

**Reply by MartinMelikMerkumians** [reply]
Hi.
You tried it according to the provided README? If yes, please specify what is unclear

[a comment by an automated account is not shown]
